course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -120,49 +120,76 @@
                 .finally($scope.updateProgressBar);
         };
         $scope.addPeopleToCourse = function(searchTerms) {
+
             /* looks up HUIDs, XIDs, and/or email addresses from searchTerms
              and attempts to add people to the course who do not already have an
              enrollment.
              */
-            var membersByUserId = {};
-            $scope.clearMessages();
-            $scope.operationInProgress = true;
-            var searchTermList = $scope.getSearchTermList(searchTerms);
-            $scope.tracking.total = searchTermList.length;
-            $scope.updateProgressBar('Looking up ' + $scope.tracking.total
-                + ' people');
-            var memberPromise = $scope.lookupCourseMembers()
-                .then(function updateCourseMembers(memberResponse) {
-                    membersByUserId = $scope.getMembersByUserId(
-                        memberResponse.data.results);
-                    return memberResponse;
-                }, function courseMemberLookupFailed(memberResponse) {
-                    $scope.handleAjaxErrorResponse(memberResponse);
-                    $scope.messages.warnings.push({
-                        type: 'courseMemberLookupFailed'
+
+            // split the list to get the number of people the user is trying to add
+            $scope.numPeople = searchTerms.split(',').length;
```

Comment:
`$scope.getSearchTermList(searchTerms)` encapsulates this (see line 155, below), and includes splitting by newlines as well. We can move that up to here instead.